anime-list-master.xml

```diff
@@ -41162,6 +41183,9 @@
   </anime>
   <anime anidbid="14951" tvdbid="289909" defaulttvdbseason="4" episodeoffset="" tmdbid="" imdbid="">
     <name>Shokugeki no Souma: Shin no Sara</name>
+    <mapping-list>
```

Comment:
It's `defaulttvdbseason=4`, specials aren't mapped by default

Reply:
I just tested with empty files and you're right, HAMA isn't following its own rules. It shouldn't map the specials but it is, despite there being 6 other entries AniDB IDs associated with this TVDB ID and the entry having no mapping-list.

https://github.com/ZeroQI/Hama.bundle/blob/b2a0ac4b57c83d6b6f6c174e08e1cc6d605d2a2a/Contents/Code/AnimeLists.py#L324
```python
  # Map season 0 episodes directly to tvdb season 0 episodes
  # On condition of being the only anidb id mapped to the tvdbid, its set to season 1, and has no special mappings
```

Reply:
> ```python
> 
>   # Map season 0 episodes directly to tvdb season 0 episodes
> 
>   # On condition of being the only anidb id mapped to the tvdbid, its set to season 1, and has no special mappings
> 
> ```

Well, from my perspective, this series doesn't meet this condition because it has multiple AniDB entries.

I cannot make any comment about the referenced code now. I need to refresh my Python skills 😅